OsmAnd/src/net/osmand/plus/wikivoyage/data/TravelObfHelper.java

```diff
@@ -1112,6 +1119,14 @@ public boolean publish(Amenity amenity) {
 												String tag = key.replaceFirst(GPX_EXTENSION_TAG_PREFIX, "");
 												String val = amenity.getAdditionalInfo(key);
 												gpxFileExtensions.put(tag, val);
+											} else if (key.startsWith(POINTS_GROUPS_TAG_PREFIX)) {
+												final String DELIMITER = "~~~";
```

Comment:
DELIMITER in generation and Android must be same constant (Kotlin lib)

Reply:
moved to PointsGroups in Kotlin